.ci/pitest-suppressions/pitest-filters-suppressions.xml

```diff
@@ -216,6 +216,24 @@
     <lineContent>tagMessageRegexp = null;</lineContent>
   </mutation>
 
+  <mutation unstable="false">
+    <sourceFile>SuppressWithNearbyTextFilter.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.filters.SuppressWithNearbyTextFilter$Suppression</mutatedClass>
+    <mutatedMethod>&lt;init&gt;</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
+    <description>Removed assignment to member variable eventIdRegexp</description>
+    <lineContent>eventIdRegexp = null;</lineContent>
```

Comment:
Why can't we resolve these 2 and need to suppress them?

Reply:
https://github.com/checkstyle/checkstyle/pull/12151#discussion_r961968443

I explained it in this conversation. Let me know if you think it is reasonable to do this.

Reply:
I cannot determine if it is reasonable unless you give a reason why this can't be resolved. Suppressing just because it is experimental goes against why we enabled it to begin with.